decidim-accountability/app/views/decidim/accountability/admin/results/index.html.erb

```diff
@@ -9,12 +9,20 @@
 
       <%= export_dropdowns(query) %>
       <%= import_dropdown do %>
-        <li class="imports--component imports--results">
-          <%= link_to t("actions.import", scope: "decidim.accountability", name: t("models.result.name", scope: "decidim.accountability.admin")), new_projects_import_path if allowed_to?(:create, :result) && parent_result.nil? %>
-        </li>
-        <li class="imports--component imports--results">
-          <%= link_to t("actions.import_csv", scope: "decidim.accountability"), import_results_path if allowed_to? :create, :result %>
-        </li>
+        <%= link_to new_projects_import_path do %>
+          <% if allowed_to?(:create, :result) && parent_result.nil? %>
+            <li class="imports--component imports--results">
+              <%= t("actions.import", scope: "decidim.accountability", name: t("models.result.name", scope: "decidim.accountability.admin")) %>
+            </li>
+            <%= link_to import_results_path do %>
+              <% if allowed_to? :create, :result %>
+                <li class="imports--component imports--results">
+                  <%= t("actions.import_csv", scope: "decidim.accountability") %>
+                </li>
+              <% end %>
+            <% end %>
+          <% end %>
+        <% end %>
```

Comment:
You're doing the check for permissions inside the link, and it should be outside. You're also mixing the ends and making:
- a link
  - a check for permissions
    - another link
      - another check for permissions

One inside the other inside the other inside the other. This is how it should look like: 

```suggestion
        <% if allowed_to?(:create, :result) && parent_result.nil? %>
		      <%= link_to new_projects_import_path do %>
            <li class="imports--component imports--results">
              <%= t("actions.import", scope: "decidim.accountability", name: t("models.result.name", scope: "decidim.accountability.admin")) %>
            </li>
          <% end %>
        <% end %>
        <% if allowed_to? :create, :result %>
        	<%= link_to import_results_path do %>
            <li class="imports--component imports--results">
              <%= t("actions.import_csv", scope: "decidim.accountability") %>
            </li>
          <% end %>
        <% end %>
```